### PR TITLE
Copyedits on Same-Origin Policy in Alloy chapter

### PR DESCRIPTION
**Pull request by amyrbrown**
Here are my edits for this chapter. There are a few comments on the Files Changed page, but they're mainly typesetting notes for @MichaelDiBernardo. 

Also:
On line 262, perhaps the electronic versions of the chapter could link to the Appendix here?
On line 1055 there is a footnote to be typeset.

And I wonder if it would be worthwhile to set the Appendix in slightly smaller text, to set it off from the main text of the chapter? Depending how much work it would be. :)

### CODE REVIEW

File: same-origin-policy/same-origin-policy.markdown [hunk 1]
```diff
@@ -745,7 +745,7 @@ assert Confidentiality {
 }
 ```
 
-The integrity property is the dual of confidentiality: 
+The integrity property is the double of confidentiality:
```
Comment (amyrbrown) [inline same-origin-policy/same-origin-policy.markdown]: I wasn't entirely sure what was meant here, but if you mean "the twin of" or "the integrity property is just like confidentiality", then "double" works better than "dual".

File: same-origin-policy/same-origin-policy.markdown [hunk 2]
```diff
@@ -1222,13 +1222,13 @@ inject bad content as part of a `PostMessage` [cite PostMessage
 study].
```
Comment (amyrbrown) [inline same-origin-policy/same-origin-policy.markdown]: Flagging that we are still missing this citation.

File: same-origin-policy/same-origin-policy.markdown [hunk 3]
```diff
@@ -1308,7 +1308,7 @@ Why would a developer ever use the wildcard? It turns out that
 specifying the allowed origins can be tricky, since it may not be
```
Comment (amyrbrown) [inline same-origin-policy/same-origin-policy.markdown]: There's a citation reference at line 1305 that's still missing.

File: same-origin-policy/same-origin-policy.markdown (hunk 1 above)
Comment (eskang) [inline same-origin-policy/same-origin-policy.markdown]: In this context, we mean "integrity" is a concept that mirrors "confidentiality", so "dual" is more appropriate.

**Comment by MichaelDiBernardo** [comment]
Merging this for now -- @eskang et al will make changes in a subsequent PR, and I'll finish things up after that re: any links / refs I missed the first time around.

**Pull request merged**